components/ai_chat/core/browser/engine/engine_consumer.h

```diff
@@ -87,7 +87,7 @@ class EngineConsumer {
   // human message except page refine event.
   bool CanPerformCompletionRequest(
       const ConversationHistory& conversation_history) const;
-  int max_page_content_length_ = 0;
+  uint32_t max_page_content_length_ = 0;
```

Comment:
If we're going to use this should we use `size_t`? The chromium style guide docs seem to encourage it unless absolutely neccessary to use `uint32_t`.

Reply:
I'll read through the chromium style guide to avoid these types of mistakes in the future. Thanks!